ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/blockbody/versions/bellatrix/BeaconBlockBodyBuilderBellatrix.java

```diff
@@ -70,7 +70,11 @@ public BeaconBlockBodyBuilder executionPayloadHeader(
 
   @Override
   protected void validateSchema() {
-    checkState(schema != null || blindedSchema != null, "schema or blindedSchema must be set");
+    if (isBlinded()) {
+      checkState(blindedSchema != null, "blindedSchema must be set");
```

Comment:
to me the 'nit' was we should be doing something like
```
checkState(blindedSchema != null && schema == null, "The blindedSchema must be set, with no schema set.")
```

Reply:
@jtraglia ?

Reply:
Sorry for the slow reply. Yes, this what I originally had in mind. Thanks.